ONSdigital/eq-survey-runner
# EQ 494 Allow the user to use a confirmation page rather then a summar…

### What is the context of this PR?

Allow the user to choose if they want a summary page or a confirmation page
### How to review
1. Make sure Star Wars and MCI all still have a functioning summary page
2. Make sure 0_rouge_one.json, works and has a confirmation page
3. Using 1_0203 (easier then Star Wars as it has routing to summary) add "submission_page" : "confirmation" to the JSON and notice it has a confirmation page, take it away and see it has a summary page 
### Who worked on the PR

@LJBabbage
